### ISSUE TITLE
Add MongoExpress to docker-compose

### ISSUE THREAD

**Issue by tstorek** [N5GEH/n5geh.platform#9]
Currently, the simple docker-compsoe setup misses the mongo-express service.
Please, consider to add it as simple interface to the mongo-db. 
This often helps to understand the internal data structures of fiware.

Also consider to upgrade the version

**Comment by djs0109** [comment]
Fair point, a mongo-express can be helpful at some point. I am using the following config in my test instance:
```yml
  mongo-express:
    image: mongo-express
    restart: always
    container_name: mongo-express
    depends_on:
      - mongo-db
    environment:
      - ME_CONFIG_MONGODB_SERVER=mongo-db
    ports:
      - "8081:8081"
    networks:
      - fiware
```
You can also share your config if that does not make sense to you ;)

**Comment by SBlechmann** [comment]
addressed in #10
